bin/run_dwelling.py

```diff
@@ -17,14 +17,16 @@
 
     # Timing parameters
     'start_time': dt.datetime(2018, 1, 1, 0, 0),  # year, month, day, hour, minute
-    'time_res': dt.timedelta(minutes=10),         # time resolution of the simulation
```

Comment:
@kendallbaertlein: We'll want to revert these changes, they were just for testing on your local branch. I don't think we actually want anything in run_dwelling.py to change here.